content/en/docs/tasks/administer-cluster/separate-mount-namespace.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "Managing a Separate Mount Namespace for Kubernetes"
+linkTitle: "Separate Mount Namespace"
+weight: 100
+description: >-
+     Discussion and instructions on how to administer a Kubernetes system with all container-specific mountpoints in a separate mount namespace from the parent system
+---
+
+# Managing a Separate Mount Namespace for Kubernetes
```

Comment:
The site renderer adds a heading automatically; we don't need this one here so I would omit it.
(to preview locally: `make container-image && make container-serve`, then visit http://localhost:1313/)